src/templates/dock.html

```diff
@@ -1,7 +1,7 @@
 <div class="jw-dock jw-reset">
     {{#each this}}
-    <div class="jw-dock-button jw-background-color jw-reset" button="{{id}}">
-        <div class="jw-icon jw-dock-image jw-reset" style="background-image: url({{img}})"></div>
+    <div class="jw-dock-button jw-background-color {{#if class}}{{class}}{{/if}} jw-reset" button="{{id}}">
```

Comment:
Total nitpick here, but I usually prefer to put the extra space inside of the {{if}}